fairlearn/utils/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
we don't need a new package for this! let's just move this into the top module, so we only have `fairlearn.show_versions`. See: https://scikit-learn.org/stable/modules/generated/sklearn.show_versions.html

Reply:
They have `utils`, but they hide that fact by making it available in the top level module https://github.com/scikit-learn/scikit-learn/blob/95d4f0841/sklearn/utils/_show_versions.py#L75
I don't particularly care, so I've moved it into fairlearn/